pkg/odo/cli/catalog/util/util.go

```diff
@@ -69,12 +69,12 @@ func DisplayClusterServiceVersions(csvs *olm.ClusterServiceVersionList) {
 	log.Info("Operators available in the cluster")
 	fmt.Fprintln(w, "NAME", "\t", "CRDs")
 	for _, csv := range csvs.Items {
-		fmt.Fprintln(w, csv.ObjectMeta.Name, "\t", csvOperators(csv.Spec.CustomResourceDefinitions))
+		fmt.Fprintln(w, csv.ObjectMeta.Name, "\t", CsvOperators(csv.Spec.CustomResourceDefinitions))
 	}
 	w.Flush()
 }
 
-func csvOperators(crds olm.CustomResourceDefinitions) string {
+func CsvOperators(crds olm.CustomResourceDefinitions) string {
```

Comment:
Why are we making this function exported? Please add a comment if it needs to stay exported.

Reply:
yes, I will add a comment, it's need to be exported